### PR TITLE
Remove yarn install part and some polish

### PR DESCRIPTION
**Pull request by pascalgrimaud**

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -124,42 +121,13 @@ module.exports = class extends BaseGenerator {
         }
     }
 
-    install() {
```
Comment (pascalgrimaud) [inline generators/app/index.js]: This module doesn't change the Angular files -> so no need to launch yarn install or npm instal

File: generators/app/index.js [hunk 2]
```diff
@@ -124,42 +121,13 @@ module.exports = class extends BaseGenerator {
...
-        }
-    }
-
     end() {
         this.log('End of spring-cloud-stream generator');
     }
 
     installRabbitMq() {
         const STREAM_RABBIT_VERSION = '1.3.0.RELEASE';
-        const STREAM_CLOUD_DEPENDENCY_VERSION = 'Ditmars.RELEASE';
```
Comment (pascalgrimaud) [inline generators/app/index.js]: It's a wrong verion here, isn't it ?

Comment (antarus) [reply generators/app/index.js]: I take this version on Spring documentation, I don't check the version use in jhipster..

Comment (pascalgrimaud) [reply generators/app/index.js]: Oh, the version Ditmars really exists, I changed to the same version as JHipster: Chelsea
So here, I don't know which version we should use?

Comment (antarus) [reply generators/app/index.js]: stay consistent with the jhipster version :)

**Pull request merged**